hw/bsp/stm32f3discovery/include/bsp/bsp.h

```diff
@@ -32,7 +32,14 @@ extern "C" {
 #define bssnz_t         sec_bss_nz_core
 
 /* LED pins */
-#define LED_BLINK_PIN   (72)
```

Comment:
Keep define for LED_BLINK_PIN; given that blinky demo app uses this, we should keep that buildable for all platforms.
Maybe point LED_BLINK_PIN to LED_BLINK_PIN_1?